src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(/src/images/imac.jpeg);
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+    margin-left: 20px;
+  }
+
+  &__name {
+    width: 166px;
+    height: 36px;
+    padding: 40px 16px 0;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
```

Comment:
The `width` and `height` properties should not be used for text containers. Instead, use padding and allow the text to define its own height. This approach is more flexible and maintainable.